templates/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 
 <head>
-    <title>Setup Corn</title>
+    <title>Setup Reservation System</title>
```

Comment:
Do we want to make this configurable as well?

Reply:
Added page title to config file